api/controllers/build-task.js

```diff
@@ -137,20 +119,32 @@ module.exports = wrapHandlers({
       // return identical responses for missing tasks and unauthorized tasks
       return res.notFound();
     }
-
     const taskJSON = buildTaskSerializer.serialize(task);
 
     // add report data to the task
     let report = null;
-    const key = `${task.artifact}${subPage || 'index'}.json`;
-    const reportReponse = await getObject(task.Build.Site, key);
-    const reportString = await reportReponse.Body.transformToString();
-    report = JSON.parse(reportString);
+    // for local seeded reports
+    if (process.env.FEATURE_LOCAL_BUILD_REPORTS === 'active') {
+      const localTask = localSiteBuildTasks.find(t => t.id.toString() === taskId);
+      const file = subPage || 'index';
+      const reportPath = join(
+        __dirname,
+        `../../services/local/tasks/${localTask.type}/${file}.json`
+      );
+
+      const raw = await readFile(reportPath, 'utf-8');
+      report = JSON.parse(raw);
+    } else {
+      const key = `${task.artifact}${subPage || 'index'}.json`;
+      const reportReponse = await getObject(task.Build.Site, key);
+      const reportString = await reportReponse.Body.transformToString();
+      report = JSON.parse(reportString);
+    }
 
     const type = appMatch(task.BuildTaskType);
 
     const fullJSON = {
-      ...taskJSON, report, type, siteId: task.Build.Site.id,
+      ...taskJSON, report, type, siteId: task.Build.Site.id, siteBuildTaskId: task.SiteBuildTask.id,
```

Comment:
`siteBuildTaskId` doesn't need to be added here now that it's been added to the serializer/json/include